nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/DeleteHDFS.java

```diff
@@ -139,6 +137,7 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
         // If this processor has an incoming connection, then do not run unless a
         // FlowFile is actually sent through
         if (originalFlowFile == null && context.hasIncomingConnection()) {
+            context.yield();
```

Comment:
I don't think we need to add the `yield()` back right?

Since we are removing `@TriggerWhenEmpty` the only way we can now get a null flow file would be a concurrent task execution where one task executes the processor at a split second where another task executed and grabbed the flow file. In this case we don't need/want to yield.

Reply:
That makes sense.. going to remove it